charts/lws/templates/certmanager/certificatemetrics.yaml

```diff
@@ -0,0 +1,18 @@
+{{- if .Values.enableCertManager }}
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  labels:
+    app.kubernetes.io/instance: serving-cert
+    {{- include "lws.labels" . | nindent 4 }}
+  name: lws-metrics-cert
```

Comment:
why do we have two metrics certs?

Reply:
There 2 different types of configurations. Kustomize prepends `lws` to the resource names but helm does not.

Reply:
Makes sense now, just noticed the helm support. Added a helm support to https://github.com/kubernetes-sigs/jobset/pull/863 as well.